docs/overview/concepts/peer-exchange.md

```diff
@@ -2,7 +2,7 @@
 title: Peer Exchange
 ---
 
-The primary objective of this protocol is to facilitate peer connectivity for resource-limited devices. The peer exchange protocol enables lightweight nodes to request peers from other nodes within the network. Light nodes can bootstrap and expand their mesh independently without relying on `Discv5`. Please refer to the [Peer Exchange](https://rfc.vac.dev/spec/34/) specification to learn more.
+The primary objective of this protocol is to facilitate peer connectivity for resource-limited devices. The peer exchange protocol enables light nodes to request peers from other nodes within the network. Light nodes can bootstrap and expand their mesh independently without relying on `Discv5`. Please refer to the [Peer Exchange](https://rfc.vac.dev/spec/34/) specification to learn more.
```

Comment:
Perhaps important to highlight here that this allows requesting _random_ peers from other nodes within the network, without requiring those nodes to divulge anything about their own connectivity or neighborhood. This is important from security perspective.